Omise.Examples/Example.cs

```diff
@@ -6,14 +6,15 @@ namespace Omise.Examples
 {
     public abstract class Example
     {
-        public const string OMISE_PKEY = "pkey_test_5948irkcbyvkleqd18w";
-        public const string OMISE_SKEY = "skey_test_59267l89d7wijuys94e";
+        public const string OMISE_PKEY = "pkey_test_59yny1yz6bxq5gfkbf1";
+        public const string OMISE_SKEY = "skey_test_59yny2e94mwhyr1lp7h";
 
         protected Client Client { get; private set; }
 
         public Example()
         {
-            Client = new Client(OMISE_PKEY, OMISE_SKEY);
+            this.Client = new Client(OMISE_PKEY, OMISE_SKEY);
+            this.Client.APIVersion = "2017-11-02";
```

Comment:
@tonydwoodhouse Lets remove this feature and bake the version into the lib directly krub.

Reason being: Because then we 'll have to write libraries to support multiple versions at once but for merchants, they don't really need to care and we shouldn't require them to care. The API they're primarily interacting with is our .NET API anyway, not the underlying HTTP calls.

Reply:
OK sure, that makes sense, especially as new merchants will be automatically on the new version